autoscheduler/frontend/src/components/App/App.tsx

```diff
@@ -1,23 +1,43 @@
 import * as React from 'react';
-import { Router } from '@reach/router';
+import { Router, navigate } from '@reach/router';
 import { ThemeProvider } from '@material-ui/styles';
-
+import StepButton from './StepButton';
 import theme from '../../theme';
 import * as styles from './App.css';
-import NavBar from '../NavBar';
+import NavBar from '../NavBar/NavBar';
 import LandingPage from '../LandingPage/LandingPage';
-import SchedulingPage from '../SchedulingPage/SchedulingPage';
+import CourseSelectPage from '../CourseSelectPage/CourseSelectPage';
+import AvailabilityPage from '../AvailabilityPage/AvailabilityPage';
+import SchedulesPage from '../SchedulesPage/SchedulesPage';
+import CustomizeSchedulePage from '../CustomizeSchedulePage/CustomizeSchedulePage';
+import { StepContext } from '../NavBar/stepManager';
+import steps from '../NavBar/steps';
 
 const App: React.SFC = function App() {
+  const [activeStep, _setActiveStep] = React.useState(
```

Comment:
Instead of changing App.tsx, I think this should be its separate file & route so we can basically A/B test it, as I think we should really think and get a good feel w/ it over a bit of time before switching completely to a different layout if that makes sense

Reply:
Okay, I put back in the old routes and set up for A/B testing. After picking a term, the site will randomly decide whether to take them to the single-page layout or the multi-page layout. A cookie will then remember that selection so that they continue to see the same layout the next time they return.